js/map_utils.js

```diff
@@ -19,27 +21,28 @@ function addStates(map, stateData) {
     })
     .attr('d', buildPath);
 
-  var nationBounds = buildPath.bounds(stateData);
+  var nationBounds = buildPath.bounds(stateBounds);
   nationDims = {
     width: nationBounds[1][0] - nationBounds[0][0],
     height: nationBounds[1][1] - nationBounds[0][1]
   };
 
   // if URL specifies a state view, zoom to that now
   var newView = getHash('view');
-  if(newView == null) { newView = 'USA'; }
+  if(newView === null) { newView = 'USA'; }
   if(newView != 'USA') {
     updateView(newView, fireAnalytics = false);
   }
 }
 
 // Function to look up a style
-formatState = function(attr, d, active) {
+function formatState(attr, d, active) {
+  var view;
   if(activeView == 'USA') {
```

Comment:
does this need `===` instead of just two?

Reply:
yes, thanks